AudibleApi/Api.Records.cs

```diff
@@ -0,0 +1,120 @@
+﻿using AudibleApi.Common;
+using Dinah.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace AudibleApi
+{
+	public partial class Api
+	{
+		private const string FIONA_DOMAIN = "ht" + "tps://cde-ta-g7g.amazon.com";
+		private const string FIONA_SIDECAR_PATH = "/FionaCDEServiceEngine/sidecar";
+
+		public async Task<bool> CreateRecordsAsync(string asin, AnnotationBuilder annotationBuilder)
+		{
+			const string requestUri = FIONA_SIDECAR_PATH + $"?type=AUDI";
+
+			ArgumentValidator.EnsureNotNullOrWhiteSpace(asin, nameof(asin));
+			ArgumentValidator.EnsureNotNull(annotationBuilder, nameof(annotationBuilder));
+			if (annotationBuilder.Count == 0) return false;
+
+			try
+			{
+				var client = Sharer.GetSharedHttpClient(FIONA_DOMAIN);
```

Comment:
Shortly after I wrote my orig shared http client stuff, MS realized their mistake in the disposable HttpClient and created a native factory. I haven't compared it to my solution yet. I imagine theirs is better but I fear breaking the world by making the switch. Since it's so deeply embedded throughout this API